jobs/e2e_node/crio/README.md

```diff
@@ -4,7 +4,7 @@
 [@kubernetes/sig-node-cri-o-test-maintainers](https://github.com/orgs/kubernetes/teams/sig-node-cri-o-test-maintainers)
 on the related issue or pull request.**
 
-All jobs maintained within this directoy are part of the `sig-node-cri-o`
+All jobs maintained within this directory are part of the `sig-node-cri-o`
```

Comment:
this is the only reason it is marked for sig-node. consider splitting it to the different PR.